config/dummy_invalidate.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
If this file is only used for testing purposes, put it in the test fixtures folder